src/dapp/libexec/dapp/dapp-verify-contract

```diff
@@ -93,7 +95,8 @@ while [ $count -lt 5 ]; do
   response=$(curl -fsS "$ETHERSCAN_API_URL" -d "$query" \
   --data-urlencode "compilerversion=$version" \
   --data-urlencode "sourceCode@"<(echo "$source") \
-  --data-urlencode "constructorArguements=$constructorArguments" -X POST)
+  --data-urlencode "$constructorArguments && constructorArguements=$constructorArguments" \
```

Comment:
this is a noop, `&&` will always make sure the second argument executes. If you remove this change the script should work the same

Reply:
Btw there is a typo in the original code - `Arguements`.

Reply:
yes, it's etherscans api that has the typo lol

Reply:
I included that because I figured that if $constructorArguments wasn't defined, then the left hand side of `&&` would be false. 

ironically it looks like this isn't needed, since after all this PR is about having no constructor args and this is working fine